clients/python/provider/examples/sse_example.py

```
@@ -0,0 +1,59 @@
+
+ENDPOINT  = os.environ.get("SUPERPOSITION_ENDPOINT", "http://localhost:8080")
+TOKEN     = os.environ.get("SUPERPOSITION_TOKEN", "token")
+ORG       = os.environ.get("SUPERPOSITION_ORG_ID", "localorg")
+WORKSPACE = os.environ.get("SUPERPOSITION_WORKSPACE", "dev")
+
+
+async def main():
+    loop = asyncio.get_event_loop()
+    _orig = loop.default_exception_handler
+    loop.set_exception_handler(
+        lambda l, ctx: None if ctx.get("message") == "Unclosed client session" else _orig(ctx)
```

Ruff (0.15.10)

[error] Ambiguous variable name: `l`

(E741)